.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Report an issue with the InfluxData documentation.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+_Describe the issue here._
+
+##### Relevant URLs
+- _Provide relevant URLs (the doc page in question, project issues, community threads, etc.)_
+
+<!--
+For InfluxDB 2.0 documentation issues, create an issue in this repository. For project issues (bugs, unexpected behavior, etc.), create an issue in the appropriate project repository:
+
+- InfluxDB issues at https://github.com/influxdata/influxdb
```

Comment:
Maybe a note about reporting Cloud specific (1.x and 2.0?), and InfluxDB Enterprise issues?

Reply:
InfluxDB 1.x and 2.0 are generated from the same repo, just different branches. Cloud and Enterprise are closed source. The repos are private.

Reply:
Yes, my thought was given that we do document Cloud 2.0 and 1.x Enterprise in the docs-v2 repo, we should let folks know how to report issues for those products to Support team.